### PR TITLE
Use static success forward

### PR DESCRIPTION
**Pull request by alisihab**

### CODE REVIEW

File: core/src/main/java/nl/nn/adapterframework/core/PipeForward.java [hunk 1]
```diff
@@ -34,6 +34,7 @@
  */
 public class PipeForward {
 
+	public final static String SUCCESS_FORWARD = PipeLineExit.EXIT_STATE_SUCCESS;
```
Comment (gvanbrakel) [inline core/src/main/java/nl/nn/adapterframework/core/PipeForward.java:37]: I would suggest to call this `SUCCESS_FORWARD_NAME`

File: core/src/main/java/nl/nn/adapterframework/webcontrol/action/AdapterHandler.java [hunk 2]
```diff
@@ -77,6 +78,6 @@ public ActionForward executeSub(
 			saveErrors(request, errors);
 		} // Forward control to the specified success URI
 		log.debug("forward to success");
-		return (mapping.findForward("success"));
+		return (mapping.findForward(PipeForward.SUCCESS_FORWARD));
```
Comment (gvanbrakel) [inline core/src/main/java/nl/nn/adapterframework/webcontrol/action/AdapterHandler.java:81]: All the mappings in webcontrol/action/* are not PipeForwards, but ActionForwards

**Review by gvanbrakel — CHANGES_REQUESTED** [review]

**Review by gvanbrakel — APPROVED** [review]

File: core/src/main/java/nl/nn/adapterframework/ldap/LdapChallengePipe.java [hunk 3]
```diff
@@ -147,7 +148,7 @@ public PipeRunResult doPipe(Message msg, PipeLineSession pls) throws PipeRunExce
 			return new PipeRunResult(findForward("invalid"), msg);
 		}
 						
-		return new PipeRunResult(findForward("success"), msg);
```
Comment (nielsm5) [inline core/src/main/java/nl/nn/adapterframework/ldap/LdapChallengePipe.java:150]: isn't this the getSuccessForward() ?

**Review by nielsm5 — CHANGES_REQUESTED** [review]

File: core/src/main/java/nl/nn/adapterframework/ldap/LdapChallengePipe.java (hunk 3 above)
Comment (gvanbrakel) [reply core/src/main/java/nl/nn/adapterframework/ldap/LdapChallengePipe.java:150]: no, LdapChallengPipe is not based on FixedForwardPipe (I think for no good reason)

**Review by nielsm5 — APPROVED** [review]

**Pull request merged**